srcpkgs/qpdfview/template

```diff
@@ -12,8 +12,8 @@ short_desc="Tabbed document viewer"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://launchpad.net/qpdfview"
-distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-${version}.tar.gz"
-checksum=cc642e7fa74029373ca9b9fbc29adc4883f8b455130a78ad54746d6844a0396c
+distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-0.5.tar.gz"
```

Comment:
use ${version} here

Reply:
it wouldnt match then

version=**0.5.0**
file name qpdfview-**0.5**.tar.gz no **.0** here

I think better leave as is for now
then
Could be changed with new release with proper naming convention

or maybe patch for now?

Reply:
`version=0.5.0`
`distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-0.5.tar.gz"`

or

`version=0.5`
`distfiles="https://launchpad.net/qpdfview/trunk/${version}.0/+download/qpdfview-${version}.tar.gz"`

File name version number differ from upstream version number 
How to handle that?